src/messages.js

```diff
@@ -1 +1,2 @@
-export const tokenBannedMessage = ''; // TODO: this message is being disscussed here: https://github.com/HathorNetwork/hathor-explorer/issues/92
\ No newline at end of file
+export const tokenBannedMessage = ''; // TODO: this message is being disscussed here: https://github.com/HathorNetwork/hathor-explorer/issues/92
+export const txTokenBannedMessage = ''; // TODO: this message is being disscussed here: https://github.com/HathorNetwork/hathor-explorer/issues/92
```

Comment:
What's this message?

Reply:
This is to show on the `TxAlerts` component whenever a tx contains a token that was banned.